dockerfiles/depwatcher/src/depwatcher/app_dynamics_agent.cr

```diff
@@ -30,18 +31,18 @@ module Depwatcher
     end
 
     private def releases()
-      response = client.get("https://download.appdynamics.com/download/downloadfilelatest/?format=json").body
-      Array(Entry).from_json(response).map do |entry|
-        if (entry.filetype == "php-tar" && entry.os == "linux" && entry.bit == "64" && entry.extension == "tar.bz2" && !entry.is_beta)
-          Release.new(
-            entry.version,
-            "https://packages.appdynamics.com/php/#{entry.version}/appdynamics-php-agent-linux_x64-#{entry.version}.tar.bz2",
-            entry.sha256_checksum || ""
-          )
-        else
-          nil
-        end
-      end.compact.sort_by { |r| Version.new(r.ref) }.last(10)
+      allReleases = Array(Release).new
+      response = client.get("https://download.run.pivotal.io/appdynamics-php/index.yml").body
+      response.each_line do |appdVersion|
+        splitArray = appdVersion.split(": ")
+        version = splitArray[0].sub("_", "-")
+        url = splitArray[1]
+        File.write("#{version}.tar.bz2", client.get(url).body)
```

Comment:
Rather than assume `tar.bz2`, it would be best to use the file name present in the URL

Reply:
Actually, now that I think about it: since you're not doing any decompressing, you could choose any file name (or no extension even)

Reply:
Yes, It's only computing the SHA256, so any extension/name of the file should work.